third_party/qemu/README.md

```diff
@@ -1,48 +1,112 @@
 # QEMU
 
-By default, the build system will download a pre-built release of our [QEMU fork](https://github.com/lowRISC/qemu/).
-In order to support local development and testing, the build system also supports a way to build QEMU from source.
-See instructions below.
+For instructions on setting up QEMU for local development and troubleshooting steps, see the [setup guide](./setup.md).
 
-## Building from source using Bazel
+## Introduction
 
-To perform local development, you first need to check out a copy of [QEMU fork](https://github.com/lowRISC/qemu/) and switch to the correct branch.
+[QEMU](https://www.qemu.org/) is an open-source project which primarily aims to support virtualization, user-mode emulation and full-system (machine) emulation flows.
```

Comment:
```suggestion
[QEMU](https://www.qemu.org/) is an open-source emulator project for functional full-system or user-mode emulation for a variety of machines and architectures.
```
I think it is important to introduce from the outset that QEMU is a generic emulator project that attempts to do functional emulation which may not be the most accurate.